api/serializers.py

```diff
@@ -10,7 +10,7 @@ class CategorySerializer(serializers.ModelSerializer):
 
     class Meta:
         model = Category
-        fields = ['id', 'name', 'tags', 'created']
+        fields = ['id', 'name', 'tags', 'locations', 'created']
```

Comment:
Seems to be causing issues, I thought we weren't adding locations to the categories?
![image](https://user-images.githubusercontent.com/32081318/220794147-678ea747-af3a-4dd9-b4b6-4e802c986196.png)